src/templates/vault-setup-script.sh.tpl

```diff
@@ -0,0 +1,388 @@
+#!/bin/sh
```

Comment:
@ShahinMahmody nachdem das skript jetzt im generator enthalten ist wird es vlt. zeit das zu verschönern/verbessern in nächster zeit :D